app/plugins/auth-plugin.js

```diff
@@ -20,15 +20,7 @@ module.exports = {
         redirectTo: (request) => {
           return auth.requestAuthorizationCodeUrl(session, request)
         },
-        validateFunc: async (request, s) => {
-          const result = { valid: false }
-
-          if (session.getClaim(request, organisationKey) || session.getEndemicsClaim(request, organisationKey)) {
-            result.valid = true
-          }
-
-          return result
-        }
+        validateFunc: async (request, _) => ({ valid: !!session.getEndemicsClaim(request, organisationKey) })
```

Comment:
```suggestion
        validateFunc: async (request, _) => ({ valid: Boolean(session.getEndemicsClaim(request, organisationKey)) })
```
Find Boolean a bit clearer personally.